noggin/controller/user.py

```diff
@@ -36,8 +36,13 @@ def user(ipa, username):
         Group(g)
         for g in ipa.group_find(membermanager_user=username, all=False)['result']
     ]
+    hidden_groups = app.config.get('HIDDEN_GROUPS')
+    complete_group_list = [group for group in groups if group.name not in hidden_groups]
```

Comment:
You could do this filtering directly in the list comprehension that sets `groups` above.